bve-corpus/src/bin/bve-parser-run.rs

```diff
@@ -36,16 +36,16 @@ fn parse_mesh_b3d_csv(file: impl AsRef<Path>, errors: bool, b3d: bool) {
     };
 
     let start = Instant::now();
-    let mut parsed = mesh_from_str(&contents, file_type);
+    let parsed = mesh_from_str(&contents, file_type);
     let duration = Instant::now() - start;
 
     println!("Duration: {:.4}", duration.as_secs_f32());
 
-    parsed
-        .errors
-        .retain(|v| !matches!(v.kind, MeshErrorKind::UselessInstruction{..}));
-
     if errors {
```

Comment:
Do you set `errors` to true if there are warnings? If so, it should be named `has_errors_or_warnings` or something. However, I wouldn't print out errors and warnings in the same branch

Reply:
That boolean comes from the CLI interface if the user wants warnings/errors printed or not. The tool is a complete hack job to begin with, so I need to clean up it as a whole later, but it's low on the priority list.